core/src/main/java/fr/sncf/osrd/envelope_sim/allowances/AllowanceValue.java

```diff
@@ -22,6 +22,9 @@ public final double getSectionAllowanceTime(
     /** Returns the allowance, given the total time and distance of the trip */
     public abstract double getAllowanceTime(double baseTime, double distance);
 
+    /** Returns the allowance, given the total time and distance of the trip */
+    public abstract double getAllowancePercentage(double baseTime, double distance);
```

Comment:
Why percentage and not ratio? In my mind percentages are mostly used as a display unit